src/components/Calculator.js

```diff
@@ -25,220 +22,218 @@ class Calculator extends Component {
     return '0';
   };
 
-  handleClick = (btnName) => {
-    const { next, total, operation } = this.state;
+  const handleClick = (btnName) => {
+    const { next, total, operation } = calcState;
 
     const obj = {
       total,
       next,
```

Comment:
* I get this error when i perform divide by zero operations kindly fix it 🙏 
![image](https://user-images.githubusercontent.com/52497246/129310629-9956c849-cfa3-4054-8cd5-a45eb94e0637.png)

Reply:
Well resolved this! Good job..